onnx_caffe2/backend.py

```diff
@@ -197,6 +200,33 @@ def _create_constant(cls, n):
         assert len(n.outputs) == 1
         return cls._create_tensor_filling_op(n.attrs["value"], n.outputs[0])
 
+    @classmethod
+    def _create_gemm(cls, n):
+        (A, B, C) = n.inputs
+        (Y,) = n.outputs
+        alpha = n.attrs.get('alpha', 1)
```

Comment:
This is dangerous. I read `CreateOperator` and it directly uses the float-ness of a value to figure out whether or not to put the value in `i` or `f`. As written here, it will go into `i`. Will Caffe2 accept this? You never actually trigger the problem because you test if alpha/beta equal 1 and omit them entirely in those cases.

Reply:
Good catch, will fix it.